src/nuget/CodeFormatter.nuspec

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0"?>
-<package >
-  <metadata>
-    <id>CodeFormatter</id>
-    <version>1.0.0-prerelease</version>
-    <title>CodeFormatter</title>
-    <authors>Microsoft</authors>
-    <owners>Microsoft</owners>
-    <description>CodeFormatter tool that enforces default coding guidelines.</description>
-    <copyright>Copyright © Microsoft Corporation</copyright>
-    <dependencies>
-      <dependency id="Microsoft.CodeAnalysis" version="1.0.0-beta1-20141031-01" />
-      <dependency id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="1.0.0-beta1-20141031-01" />
-      <dependency id="Microsoft.CodeAnalysis.Workspaces.Common" version="1.0.0-beta1-20141031-01" />
-      <dependency id="Microsoft.Composition" version="1.0.0-beta1-20141031-01" />
-      <dependency id="System.Reflection.Metadata" version="1.0.0-beta1-20141031-01" />
-    </dependencies>
-  </metadata>
-  <files>
-    <file src="CodeFormatter.exe" target="lib\net45" />
```

Comment:
I see we had `lib\net45` here, but all the .csprojs have `net46`.  Just making sure we are OK with this change.

Reply:
They weren't even targeting 4.5 before, but actually 4.5.2.  Latest roslyn and msbuild require 4.6 so I think its OK.